grammar.y

```diff
@@ -1412,10 +1412,10 @@ insert_stmt:
         sel.OrderBy = append(sel.OrderBy, &OrderingTerm{Expr: &Column{Name: Identifier("rowid")}, Direction: AscStr, Nulls: NullsNil})
       }
 
-      $$ = &Insert{Table: $3, Columns: ColumnList{}, Rows: []Exprs{}, Select: sel, Upsert: $6}
```

Comment:
this was i bug that i found. the columns were not being passed to the ast

Reply:
good catch, I ran into this bug working on the voting project!